Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -105,10 +109,9 @@ const KeyboardAvoidingView = React.createClass({
       this.setState({bottom: 0});
       return;
     }
-
+    StatusBar.setHidden(false);//https://code.google.com/p/android/issues/detail?id=5497
```

Comment:
Can you explain a bit more here in a comment? A brief description is great, and people can look up the thread later for more context.